src/Library/UEventDeviceManager.cpp

```diff
@@ -245,21 +245,40 @@ namespace usbguard
   /*
    * Manager
    */
+  void UEventDeviceManager::setDeviceAuthorizedDefault(SysFSDevice* device, DeviceManager::AuthorizedDefaultType auth_default)
```

Comment:
It might make sense to move this to DeviceManager instead of UEventDeviceManager.